### PR TITLE
OSDOCS-5630:updates 4.13's RNs TP table

### PR DESCRIPTION
**Pull request by JoeAldinger**
[OSDOCS-5630](https://issues.redhat.com//browse/OSDOCS-5630):updates 4.13's RNs TP table

Version(s):
4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-5630

Link to docs preview:
http://file.rdu.redhat.com/jaldinge/OSDOCS-5630/release_notes/ocp-4-13-release-notes.html#ocp-4-13-technology-preview

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -2109,28 +2084,28 @@ In the following tables, features are marked with the following statuses:
...
 |Not Available
-|Not Available
+|Technology Preview
 |Technology Preview
 
 |MT2892 Family [ConnectX-6 Dx] SR-IOV support
 |Not Available
-|Not Available
+|Technology Preview
 |Technology Preview
```
Comment (wizhaoredhat) [inline release_notes/ocp-4-13-release-notes.adoc:2093]: GA

**Comment by wizhaoredhat** [comment]
LGTM

**Comment by stevsmit** [comment]
LGTM

**Review by kalexand-rh — APPROVED** [review]

**Pull request merged**